lib/jekyll-mentions.rb

```diff
@@ -2,28 +2,36 @@
 require 'html/pipeline'
 
 module Jekyll
-  class Mentions < Jekyll::Generator
+  class Mentions < Jekyll::Converter
+    MENTIONABLE_EXTS = %w{.html .htm .md .markdown .mkdn .mkd .textile}
```

Comment:
One thing we could do to start with is assume it's HTML and only block typically-non-html like Sass, Css, JS, XML, etc.

Ultimately, it would be nice to have this method get the output ext too, but that's determined by the converters so... Not super useful.